scripts/build-index.js

```diff
@@ -29,8 +29,19 @@ function buildPagesIndex(files) {
       if (!index[page].language.includes(language)) {
         index[page].language.push(language);
       }
+
+      const targets = index[page].targets;
+      const exist = targets.some((t) => {return t.platform === os && t.language === language});
+      if (!exist) {
+        targets.push({"os": os, "language": language})
+      }
     } else {
-      index[page] = {name: page, platform: [os], language: [language]};
+      index[page] = {
+        name: page,
+        platform: [os],
+        language: [language],
+        targets: [{"os": os, "language": language}]
```

Comment:
Now that I think about it this could be rewritten in ES6 style as:

```suggestion
        targets: [{os, language}]
```

What do you think?

Reply:
Ooh, I had forgotten about that feature. Nice.